package.json

```diff
@@ -28,7 +28,7 @@
     "version": "oclif-dev readme && git add README.md",
     "format": "prettier --write .",
     "check-format": "prettier --check .",
-    "pkg": "npm run prepack && pkg . --compress brotli"
+    "pkg": "npm run prepack && pkg . --compress gzip"
```

Comment:
reason to change to gzip? is it smaller?

Reply:
Here are my data for Brotli compression:
Compile time:
![image](https://user-images.githubusercontent.com/16670475/130024275-c3ea3075-f484-4296-af0e-94434f6970bb.png)
File size:
![image](https://user-images.githubusercontent.com/16670475/130024321-0796a5ad-0038-4000-a814-042a4d65afa1.png)

GZIP:
![image](https://user-images.githubusercontent.com/16670475/130024615-91b05df4-00ee-4274-9f0b-3ddc4297338c.png)
File size:
![image](https://user-images.githubusercontent.com/16670475/130024632-8a579ff0-2a47-4928-b63b-ee2462c25879.png)

In terms of filesize, there are roughly ~5MB difference. But one thing that should be noted is that the compile time is faster. It probably because of my machine, though. 

As for startup time, there is a slight difference up to ~1 seconds between running using brotli compressed and gzip compressed (Brotli is slower). But I couldn't measure the startup time, so I counted manually

Reply:
After some consideration, I changed it back to Brotli. The reason is even though the compile time is longer, it will be compiled only once and users are able to use smaller size binary.